packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/map_configuration.dart

```diff
@@ -3,8 +3,7 @@
 // found in the LICENSE file.
 
 import 'package:flutter/widgets.dart';
-
-import 'ui.dart';
+import '../../google_maps_flutter_platform_interface.dart';
```

Comment:
Nit: Our style is to leave a blank line between package and non-package dependencies.